.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "prettier"
+  ],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "plugins": [
+    "flowtype",
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": "error"
```

Comment:
My understanding is that this rule configures Eslint to be _compatible_ with Prettier styles (or maybe rather, not to _conflict_ with Prettier's preferences), but they still do different things. For example, if I add the `no-unused-expressions` rule (which I probably will in a follow-up) to Eslint's config, it will complain about variables that are declared but not used. Prettier will ignore them.